packages/core/src/modules/connections/TrustPingEvents.ts

```diff
@@ -0,0 +1,22 @@
+import type { BaseEvent } from '../../agent/Events'
+import type { DidExchangeState } from './models'
+import type { ConnectionRecord } from './repository/ConnectionRecord'
+
+export enum TrustPingEventTypes {
+  TrustPingRequestEvent = 'TrustPingRequestEvent',
+  TrustPingResponseEvent = 'TrustPingResponseEvent',
+}
+
+export interface TrustPingRequestEvent extends BaseEvent {
+  type: typeof TrustPingEventTypes.TrustPingRequestEvent
+  payload: {
+    connectionRecord: ConnectionRecord
+  }
```

Comment:
Can we also add the message to the event? That will make it possible to track based on the threadId

```suggestion
  payload: {
    connectionRecord: ConnectionRecord
    message: TrustPingMessage
  }
```

Reply:
Done